config/routes.rb

```diff
@@ -2,10 +2,11 @@
 
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  match ':controller(/:action(/:id))', :via => [:get, :post]
   namespace 'api' do
     namespace 'v1' do
-      resources :products
+      resources :products do
+        resources :product_versions
+      end
```

Comment:
Since at present we are only supporting `index` method, can we add a only clause here. Check this out: https://guides.rubyonrails.org/routing.html#restricting-the-routes-created

```
resources :products, only: [:index] do
  resources :product_versions, only: [:index]
end
```

We can add more methods to the only clause or remove it all together as need be when we add more methods to our controller in future.